nextcord/application_command.py

```diff
@@ -1265,77 +1305,176 @@ def __init__(
             cmd_arg_given = False
 
         self.name = cmd_arg.name or parameter.name
+        # Use the given name, or default to the parameter name.
+        # typehint_origin = typing.get_origin(parameter.annotation)  # TODO: Once Python 3.10 is standard, use this.
```

Comment:
These comments are unnecessary. It's pretty obvious that once Python 3.10 is standard, we replace `typing_extensions` with just `typing`.